src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -25,24 +23,22 @@ import { IXtermTerminal } from 'vs/workbench/contrib/terminal/browser/terminal';
 import type { Terminal } from 'xterm';
 
 const enum AccessibleBufferConstants {
```

Comment:
nit: I'd rename this to just `Constants` as that's the topic of the file and it's not exported